fix_compliance.py

```
@@ -0,0 +1,21 @@
+import re
+
+with open('crates/logfwd/tests/it/compliance.rs', 'r') as f:
```

Ruff (0.15.10)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)